flink-test-utils-parent/flink-test-utils-junit/src/main/java/org/apache/flink/core/testutils/AllCallbackWrapper.java

```diff
@@ -22,7 +22,11 @@
 import org.junit.jupiter.api.extension.BeforeAllCallback;
 import org.junit.jupiter.api.extension.ExtensionContext;
 
-/** An extension wrap logic for {@link BeforeAllCallback} and {@link AfterAllCallback}. */
+/**
+ * An extension wrap logic for {@link BeforeAllCallback} and {@link AfterAllCallback}.
+ *
+ * <p>You MUST not use this with {@code MiniClusterExtension}.
```

Comment:
Now that the flink-runtime MiniClusterExtension was renamed this should be caught by the compiler :)

Reply:
That's why I use the `@code` tag :)

Reply:
What I mean is that you can't use it with the CallbackWrappers in the first place, because it doesn't implement `CustomExtension`.

Reply:
You're totally right, sorry :smile: